### ISSUE TITLE
Port to Underground 1?

### ISSUE THREAD

**Issue by wefalltomorrow** [thelink2012/novisualrep#1]
Not an issue, but not sure where to ask this. Would a port to Underground 1 be possible? It would be greatly appreciated as it's preventing me from progressing through the game and I can't find anything to disable it.

**Comment by thelink2012** [comment]
I do believe it's possible but unfortunately I cannot go and implement it.
Sorry.

Best,
Denilson

Em seg, 6 de mai de 2019 10:11, wefalltomorrow <notifications@github.com>
escreveu:

**Comment by wefalltomorrow** [comment]
Sorry for the late response, I appreciate your response. It's good to know it's possible, hopefully someone makes it in the future.

**Comment by thelink2012** [comment]
Good news.

Recently, I have started playing NFSU1 and have faced the problems of its visual reputation system. So, I went on, and created a patch for it too.

Source code is already available in the repository. I have thoroughly tested it, but I'll publish the binaries in NFSCars after I finish playing the game so I can be sure it doesn't have problems. In any case, the release candidate binary can be found [here](https://drive.google.com/file/d/1xUhYO3g0zOkBpT-8cg5fvz1U0wwezd8b/view).

Thanks for your suggestion.

**Comment by wefalltomorrow** [comment]
Awesome, thanks heaps! I'll definitely use this when I play UG1 again.

May I suggest using https://nfsmods.xyz/ instead of (or in addition to) NFSCars as the latter is a dying website.

**Comment by thelink2012** [comment]
I have noticed that NFSCars is a bit broken and was wondering whether I should've uploaded somewhere else. I've found nfsmods.xyz but wasn't sure if that was the de-facto standard of the community. Good to know it's what I was looking for!

However, since the NFSCars link is already spread through the web, I'll keep that, and as I don't really want to maintain the mod in two different showrooms, I'll refrain from uploading in NFSMods.

I've finishing playing and updated the mod in NFSCars. Closing the issue now.

Once again, thanks for opening the issue.
